html-syntax/index.html

```diff
@@ -9,7 +9,7 @@
 
 
     <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png
-">
+" alt="Pikachu is as overrated as it gets.">
```

Comment:
I definitely agree, but the `alt` tag should be descriptive of what the actual image is. The main purpose of this attribute is to provide people with vision impairments or other disabilities, extra information on what's appearing on the page. Something as simple as "Pikachu" would be sufficient.

Reply:
ahhhh okay lol thank you